README.md

```diff
@@ -146,15 +166,17 @@ Currently all aggregation types are compatible with all aggregation windows. Lat
     lastNSamples := 100
     precisionSubsets := 10
     wnd2 := stats.NewWindowSlidingCount(lastNSamples, precisionSubsets)
-    
     wn3 := stats.NewWindowCumulative()
 
 
 ### To creater/register a view
+
+TODO: define "view" (link to the spec).
+
 Create a view:
 
-    myView1 = stats.NewView("/my/int64/viewName", "some description", []Keys{key1, key2}, mf, agg1, wnd1)
-    myView2 := stats.NewView("/my/float64/viewName", "some other description", []Keys{key1}, mi, agg2, wnd3)
+    myView1 := stats.NewView("/my/int64/viewName", "some description", []tags.Key{key1, key2}, mf, agg1, wnd1)
```

Comment:
have a CL that fixes this also :)

Reply:
I hope the resolution is not too hard which ever gets in first. :-)

Reply:
:)